src/spikeinterface/core/waveform_extractor.py

```diff
@@ -800,6 +800,9 @@ def select_units(self, unit_ids, new_folder=None, use_relative_path: bool = Fals
 
                 if self.has_recording():
                     self.recording.dump(new_folder / "recording.json", relative_to=relative_to)
+                else:
+                    shutil.copytree(self.folder / "recording_info", new_folder / "recording_info")
```

Comment:
I would remove the else. We should copy this file for all situation no ?

Reply:
@samuelgarcia I thought the file is copied when you dump the recording?

Reply:
I agree with @samuelgarcia 

Please remove the else and copy recording info in both cases (with and without recording)

Reply:
@samuelgarcia @alejoe91  Allright, done :)